task/upload.go

```diff
@@ -551,6 +555,7 @@ func outputLocations(outURL string, relativePath string, autoMp4s bool) ([]Outpu
 					SourceSegments:     sourceSegments,
 					TranscodedSegments: true,
 					AutoMP4:            autoMp4s,
+					GenerateMP4:        generateMp4s,
```

Comment:
Continuing the comment I made on the API, I think here and on the task object the field makes sense! Only on the assets that I think it is a bit weird. 

e.g. `GET /asset/:id => { ..., generateMp4s: true }` what does it mean on a response for example? Since we're using a verb it'd have to be in the past to make more sense, but my suggestion is to not use a verb at all (on assets), using a "state description" instead.